docs/userguide/modules/creating-service-bindings/pages/binding-options.adoc

```diff
@@ -186,10 +186,14 @@ spec:
 
 == Binding workloads from a label selector
 
-Sometimes, it may be useful to specify the workload being bound by using a label selector.  For
-instance, you may want to bind a service to every workload with the label `environment: production`
-set.  SBO is able to support this kind of binding by being able to bind with label selectors.  As an
-example, to project a single secret into multiple workloads, set up your application field like so:
+You can use a label selector to specify the workload that is being bound. If you declare a service
+binding using these label selectors to pick up workloads, the {servicebinding-title} periodically
+attempts to find and bind new workloads that match the given label selector.
+
+For example, you may want to bind a service to every application with the environment: production label set in the application field. Doing so projects a single secret into multiple workloads.
```

Comment:
```suggestion
For example, you may want to bind a service to every application with the `environment: production` label set in the `application` field. Doing so projects a single secret into multiple workloads.
```

Reply:
```suggestion
For example, you may want to bind a service to every `Deployment` in a namespace with the `environment: production` label.  Setting an appropriate label selector, the {servicebinding-title} can bind each of these workloads with one `ServiceBinding` resource.
```

Thoughts?  I think this gets the intention across a little better.

Reply:
> Thoughts? I think this gets the intention across a little better.

Good. This one sounds better. YAY!! just one change: "When you set an appropriate label selector, the {servicebinding-title} can bind each of the workloads with one `ServiceBinding` resource."

Reply:
Fixed.

Reply:
@sadlerap I am not seeing where you have fixed this change. Can you please check once and let me know? Thanks!